easybuild/easyconfigs/f/fastp/fastp-0.20.0-GCCcore-8.3.0.eb

```diff
@@ -0,0 +1,38 @@
+easyblock = 'ConfigureMake'
+
+name = 'fastp'
+version = '0.20.0'
+
+homepage = 'https://github.com/OpenGene/fastp'
+description = """A tool designed to provide fast all-in-one preprocessing for FastQ files.
+ This tool is developed in C++ with multithreading supported to afford high performance."""
+
+toolchain = {'name': 'GCCcore', 'version': '8.3.0'}
```

Comment:
@Darkless012 Please make this `GCC` (and drop the `binutils` build dep).

`fasp` is likely performance sensitive, so installing it with `iccifort` when the need arises may be useful (and `GCC` is the equivalent of `iccifort` in `foss`)